ghost/core/core/server/services/email-service/EmailRenderer.js

```diff
@@ -283,6 +283,10 @@ class EmailRenderer {
         return null;
     }
 
+    getFallbackDomain() {
```

Comment:
Should this exist here, or somewhere else? When implementing this it felt like a natural place to put it, but is it correct semantically?